sunscreen_zkp_backend/src/jit.rs

```diff
@@ -329,7 +329,8 @@ where
                 let parents = query.get_unordered_operands(id)?;
 
                 for parent in parents {
-                    if node_outputs[&parent].clone().zkp_into() != x {
+                    let actual = node_outputs[&parent].clone().zkp_into();
```

Comment:
Can revert this. Moved to a new variable for debugging. Can't wait until we get an intern to make a debugger for this dumb thing.